prow_config.yaml

```diff
@@ -77,14 +74,25 @@ workflows:
     include_dirs:
       - components/centraldashboard/*
       - releasing/releaser/components/centraldashboard.jsonnet
+  # The postsubmit run publishes the docker images to gcr.io/kubeflow-images-public
+  - app_dir: kubeflow/kubeflow/releasing/releaser
+    component: centraldashboard
+    name: dashboard-release
+    job_types:
+      - postsubmit
+    params:
+      cluster: "kubeflow-testing"
```

Comment:
Nit here and below. Only set the parameters that we want to override. So in this case I think we only want to set registry. For everything else we can use the defaults.